deploy/kcp-hacbs-workspace-init/templates/service.yaml

```diff
@@ -7,6 +7,7 @@ spec:
   - name: http
     port: 80
     targetPort: 8080
+    protocol: TCP
```

Comment:
when I tried deploying this in a hacbs user workspace in cps stable it complained about the protocol not being set on the service